src/components/SelectDates/SelectDates.jsx

```diff
@@ -194,7 +194,8 @@ class SelectDates extends PureComponent {
     }
   }
 
-  onChange = value => {
+  onChange = originalValue => {
+    let value = originalValue
```

Comment:
```suggestion
    const value = originalValue
```

Reply:
No. value can be changed https://github.com/cozy/cozy-banks/blob/5c8b7407d7d35f9c70d8ef95db7f9985a683cc0e/src/components/SelectDates/SelectDates.jsx#L215